### PR TITLE
Refactored the project to include commands

### PR DESCRIPTION
**Pull request by cracker4o**
Added command classes and created a way to dynamically match commands.
When adding new commands we just need to specify the command string and create a constructor.
The bot.js will automatically figure out which command to use by dynamically loading the appropriate command based on the command string.

### CODE REVIEW

File: src/commands/help-command.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { MessageEmbed } from 'discord.js';
+
+export class HelpCommand {
+    static getCommand() {
+        return 'help';
```
Comment (cracker4o) [inline src/commands/help-command.js:5]: Every command should have a static method called getCommand which should return the command string.

File: src/commands/help-command.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    static create(message, logger) {
+        return new HelpCommand(message, logger);
```
Comment (cracker4o) [inline src/commands/help-command.js:9]: Every command should have a static constructor that should accept a Discord message object and a logger.

File: src/commands/help-command.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    constructor(message, logger){
+        this.regex = /^!mw (help|commands)(.*)?$/;
```
Comment (typat) [inline src/commands/help-command.js:13]: do we still need this regex?

**Review by typat — APPROVED** [review]
nice, looks good. we should figure out a way to move the get user stats into a commands class

File: src/commands/help-command.js (hunk 3 above)
Comment (cracker4o) [reply src/commands/help-command.js:13]: Not really. I'll remove it.

**Pull request merged**